docs/lib/duckdb.md

```diff
@@ -140,7 +140,12 @@ SELECT ST_Area('POLYGON((0 0, 0 1, 1 1, 1 0, 0 0))'::GEOMETRY) as area;
 
 To tell which extensions have been loaded, you can run the following query:
 
-```sql echo
+```js
+await new Promise((r) => setTimeout(r, 2000));
```

Comment:
Instead of waiting an arbitrary timeout, how about waiting a query that is known to autoload the JSON extension?

Reply:
fixed with `WHERE JSON '1'`